build/KoreBuild.ps1

```diff
@@ -119,15 +119,22 @@ if (!($env:Path.Split(';') -icontains $dotnetLocalInstallFolder))
 
 $makeFileProj = "$PSScriptRoot/KoreBuild.proj"
 $msbuildArtifactsDir = "$repoFolder/artifacts/msbuild"
-$msbuildLogFilePath = "$msbuildArtifactsDir/msbuild.binlog"
 $msBuildResponseFile = "$msbuildArtifactsDir/msbuild.rsp"
 
+$msBuildLogArgument = ""
+
+if ($env:KOREBUILD_ENABLE_BINARY_LOG -eq "1")
+{
+    Write-Host "Enabling binary logging because KOREBUILD_ENABLE_BINARY_LOG = 1"
+    $msbuildLogFilePath = "$msbuildArtifactsDir/msbuild.binlog"
+    $msBuildLogArgument = "/bl:$msbuildLogFilePath"
```

Comment:
This will cause perf issues but we are not using it everywhere anymore. Using `/logger` here would be a work-around (which needs to be removed when msbuild fixes the perf issue). As only small set of cases we are going to generate binary log, perhaps those machines can take the perf impact. If we find a compelling reason that we need work-around, we will update to `/logger`.
Thanks for all the help. 😄